package.json

```diff
@@ -67,6 +73,23 @@
     "nyc": "^15.1.0",
     "shelljs": "^0.8.4",
     "std-mocks": "^1.0.1",
-    "strip-ansi": "^6.0.0"
+    "strip-ansi": "^6.0.0",
+    "tsup": "^6.7.0",
+    "typescript": "^5.0.3"
+  },
+  "tsup": {
```

Comment:
I've not heard of tsup before - though from the documentation it looks solid. What benefits is it bringing here?

Reply:
Its the easiest way to build a TS library. It outputs both ESM and CJS versions. I used it to build github.com/alecvision/borg

Reply:
I'll go ahead and resolve this, but feel free to reopen it if you have a different build strategy in mind.